src/main/java/no/sikt/sws/PrefixStripper.java

```diff
@@ -62,39 +64,51 @@ public static String body(OpenSearchCommandKind commandKind,
             case ACK:
                 return responseBody.replaceAll(regex, EMPTY_STRING);
             case ERROR:
-                return Builder.errorFromValues(workspacePrefix,responseBody);
+                return ErrorDto
+                    .fromResponse(responseBody)
+                    .strippedResponse(workspacePrefix);
             case CONTENT:
                 switch (commandKind) {
                     case ALIAS:
                     case MAPPING:
-                        return responseBody.replaceAll(regex, EMPTY_STRING);
+                        return IndexDto
+                            .fromResponse(responseBody)
+                            .strippedResponse(workspacePrefix);
                     case DOC:
-                        return Builder.docFromValues(workspacePrefix,responseBody);
+                        return DocDto
+                            .fromResponse(responseBody)
+                            .strippedResponse(workspacePrefix);
                     default:
-                        throw new IllegalStateException("Unexpected value: " + commandKind);
+                        throw new IllegalStateException(UNEXPECTED_VALUE + commandKind.name());
                 }
             case CONTENT_COLLECTION:
                 switch (commandKind) {
                     case BULK:
                     case MAPPING:
-                        return responseBody.replaceAll(regex, EMPTY_STRING);
+                        return responseBody.replaceAll(regex, EMPTY_STRING);    // stripping index names in body.
```

Comment:
Was this comment intented to be pushed?

Reply:
Yes, I felt it wasn't obvious what it did.